bulletin/index.html

```diff
@@ -117,22 +117,23 @@ <h4 class="buarticle-subheading">MACILE Spanish currilum</h4>
 			</article>
 			<article class="buarticle">
 				<img src="/images/bulletin/busLeaders-2008-m.jpg">
-				<p>Visit from a Dominican business leader and a government official. They spoke about the job market and the government scholarship program. Students learned about opportunities and requirements for success.</p>
+				<p>Visit from a Dominican business leader and a government official. They spoke about the job markets and the government scholarships program. Students learned about opportunities and requirements for success.</p>
 			</article>
 			<article class="buarticle">
 				<img src="/images/bulletin/tdi-2008-math_3.jpg">
-				<h4 class="buarticle-heading">Teacher Development</h4>
+				<h3 class="buarticle-main--title">Teacher Development</h3>
```

Comment:
Again, keep your commits single purpose. The commit message says "complete content for the scholarship section" but the commit includes

- changes to class
- changes to html that does not correspond to the scholarship section

You should get in the habit of using `git add -p` to selectively stage the commits that you want. VSCode has a lovely git plugin that lets you easily pick the sections you want to add to your commit.

Reply:
Thanks for the tip. Will keep in this mind. Important to maintain the consistency of the process throughout.